lib/global.neon

```diff
@@ -142,6 +142,14 @@ DECLARE EXCEPTION Utf8Encoding
  |%
 DECLARE EXCEPTION ValueRange
 
+%|
+ |  Exception: StringIndex
```

Comment:
Make sure you keep the items in this file in the same (overall) alphabetical order.